app/models/form/lettings/questions/reason.rb

```diff
@@ -16,114 +15,217 @@ def initialize(id, hsh, page)
+          "value" => "End of social housing tenancy - evicted due to rent arrears",
+        },
+        "53" => {
+          "value" => "End of social housing tenancy - evicted for any other reason",
+        },
+        "1" => {
+          "value" => "Permanently decanted from another property owned by this landlord",
+        },
+        "2" => {
+          "value" => "Left home country as a refugee",
+        },
+        "45" => {
+          "value" => "Discharged from prison",
+        },
+        "46" => {
+          "value" => "Discharged from long-stay hospital or similar institution",
+        },
+        "4" => {
+          "value" => "Loss of tied accommodation",
+        },
+        "9" => {
+          "value" => "Asked to leave by family or friends",
+        },
+        "8" => {
+          "value" => "Relationship breakdown (non-violent) with partner",
+        },
+        "44" => {
+          "value" => "Death of household member in last settled accommodation",
+        },
+        "16" => {
+          "value" => "To move nearer to family, friends or school",
+        },
+        "17" => {
+          "value" => "To move nearer to work",
+        },
+        "48" => {
+          "value" => "Domestic abuse - previously joint tenancy with partner",
+        },
+        "49" => {
+          "value" => "Domestic abuse - other",
+        },
+        "10" => {
+          "value" => "Racial harassment",
+        },
+        "31" => {
+          "value" => "Hate crime",
+        },
+        "11" => {
+          "value" => "Other problems with neighbours",
+        },
+        "34" => {
+          "value" => "Repossession",
+        },
+        "54" => {
+          "value" => "Could no longer afford rent or mortgage",
+        },
+        "12" => {
+          "value" => "Property unsuitable because of overcrowding",
+        },
+        "13" => {
+          "value" => "Property unsuitable because of ill health or disability",
+        },
+        "14" => {
+          "value" => "Property unsuitable because of poor condition",
+        },
+        "29" => {
+          "value" => "Under occupation (offered incentive to downsize)",
+        },
+        "30" => {
+          "value" => "Under occupation (no incentive)",
+        },
+        "18" => {
+          "value" => "To move to accommodation with support",
+        },
+        "19" => {
+          "value" => "To move to independent accommodation",
+        },
+        "20" => {
+          "value" => "Other",
+        },
+        "28" => {
+          "value" => "Don’t know",
+        },
+        "divider" => {
+          "value" => true,
+        },
+        "47" => {
+          "value" => "Tenant prefers not to say",
+        },
+      }.freeze
+    else
+      {
+        "40" => {
+          "value" => "End of assured shorthold tenancy (no fault)",
+        },
+        "41" => {
+          "value" => "End of assured shorthold tenancy (eviction or tenant at fault)",
+        },
```

Comment:
Picky, but could we format this as 1 option per line?
```
...
"40" => { "value" => "End of assured shorthold tenancy (no fault)" },
"41" => { "value" => "End of assured shorthold tenancy (eviction or tenant at fault)" },
...
```

Reply:
you bet, that'll be much nicer